### PR TITLE
add express

### PR DESCRIPTION
**Pull request by itumo147**
I setup express backend for devs

### CODE REVIEW

File: backend/.gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+node_modules
```
Comment (Profsain) [inline backend/.gitignore]: check this.
node_modules folder not ignored

**Review by Profsain — CHANGES_REQUESTED** [review]
Remove node_modules folder.

**Review by missionctrlbiz — APPROVED** [review]

**Comment by missionctrlbiz** [comment]
I asked to request the review by updating the status although I have approved it

**Review by missionctrlbiz — APPROVED** [review]

**Review by missionctrlbiz — APPROVED** [review]

**Review by missionctrlbiz — COMMENTED** [review]
Added Express and cors

**Pull request merged**